pkg/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -885,17 +885,11 @@ func getLabels() map[string]string {
 
 func (v *vpnSeedServer) getEnvoyConfig() string {
 	var (
-		listenAddress = "0.0.0.0"
-		// we don't want to use AUTO for single-stack clusters as it causes an unnecessary failed lookup
-		// ref https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto#enum-config-cluster-v3-cluster-dnslookupfamily
-		dnsLookupFamily = "V4_ONLY"
```

Comment:
What does this change means with regards to DNS lookups? Will we have both queries for `A` and `AAAA` records in parallel or sequentially? What is actually looked up in the `vpn-seed-server` envoy container?

Reply:
How long will this take? This could be a degradation in service quality.

Still, I do not understand what DNS queries would actually be affected by this. Do you know?